src/model/mod.rs

```diff
@@ -474,8 +520,8 @@ mod tests {
     #[test]
     fn test_empty_control_pileup() {
         let variant = Variant::Deletion(3);
-        let tumor_all = AlleleFreq(0.0)..AlleleFreq(1.0);
-        let tumor_alt = AlleleFreq(0.0)..AlleleFreq(1.0);
+        let tumor_all = AFRange::inclusive( AlleleFreq(0.0)..AlleleFreq(1.0) );
+        let tumor_alt = AFRange::inclusive( AlleleFreq(0.0)..AlleleFreq(1.0) ); // TODO: should this be left_exclusive() instead?
```

Comment:
Yes. Numerically it does not matter in my case, but now that we are able to express this, it should be done.